index.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Webxdc. A Web Without Platforms.</title>
+    <title>webxdc. a web without platforms.</title>
```

Comment:
```suggestion
    <title>webxdc - a web without platforms</title>
```

maybe leave out the dots here as well.